suzieq/poller/worker/services/service.py

```diff
@@ -761,7 +761,7 @@ async def run(self):
                 await self.commit_data(result, output[0]["namespace"],
                                        hostname)
             else:
```

Comment:
Just a small comment:
```suggestion
            elif  self.run_once in ["gather", "process", "update"]:
```

Reply:
Ack